packages/core/src/BackgroundImage.js

```diff
@@ -1,30 +1,46 @@
-import styled from 'styled-components'
+import styled, { css } from 'styled-components'
 import PropTypes from 'prop-types'
 import { width } from 'styled-system'
-import { getPaletteColor } from './utils'
+import { applyVariations, getPaletteColor, mapProps } from './utils'
+
+const variations = {
+  parallax: css`
+    background-attachment: fixed;
+  `,
+  static: css`
+    background-attachment: scroll;
+  `
+}
 
 const image = props =>
   props.image ? { backgroundImage: `url(${props.image})` } : null
 
 const height = props => (props.height ? { height: props.height } : null)
 
-const BackgroundImage = styled.div`
+const BackgroundImage = mapProps(({ ...props }) => ({
+  ...props
+}))(styled.div`
   background-position: center;
   background-size: cover;
   background-repeat: no-repeat;
   background-color: ${getPaletteColor('border.light')};
   ${image} ${height} ${width};
-`
-const numberStringOrArray = PropTypes.oneOfType([
-  PropTypes.number,
-  PropTypes.string,
-  PropTypes.array
-])
+  ${applyVariations('BackgroundImage', variations)}
+`)
+
+BackgroundImage.defaultProps = {
+  variation: 'static'
+}
 
 BackgroundImage.propTypes = {
-  /** background-image url */
+  backgroundImage: PropTypes.string,
```

Comment:
what's this for?

Reply:
I inferred from the comment that this snippet was missing a proptype for `backgroundImage` (used above in this file) but I misread it. Not sure what the comment is for but I added it back and added a prop type for `height` instead!